doc/usecases/core/UC_keyset_lookup_direct.md

```diff
@@ -0,0 +1,24 @@
+# Use Case: Direct lookup in `KeySet`
```

Comment:
I don't think this needs an extra use case. IMHO it is simply a scenario of the lookup.

Reply:
Like I said, IMO it depends on whether the links handling is part of the core. If it is, I find it easier to separate them, because a direct lookup and a cascading lookup are a bit different in nature.